tasks/client.yaml

```diff
@@ -1,25 +1,154 @@
 ---
-- name: Operating system dependencies
-  apt: name={{ item }} state=present
+
+# dependencies
```

Comment:
I guess the user needs to copy the vars from the correct file? Might need a readme entry for that.

Btw, we could look up machine facts to see whether it's Ubuntu, Debian or RPM based - though I have no idea how reliable that would be - especially the third.